### PR TITLE
Week 3 changes (Step 1 and 2: Servlets & Fetch)

### PR DESCRIPTION
**Pull request by andyp21**
- change DataServlet.java's doGet() function to call "Hello Andrew"
- add button to index page of portfolio which uses Javascript code to fetch data from server

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,16 +12,14 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
+// Adds "Hello Andrew" to page 
+async function getGreetingAsyncAwait() {
+  const response = await fetch('/data');
+  const greeting = await response.text();
+  document.getElementById('greeting-container').innerText = greeting;
```
Comment (735952) [inline portfolio/src/main/webapp/script.js:21]: duplicate logic?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,16 +12,14 @@
...
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
+// Adds "Hello Andrew" to page 
+async function getGreetingAsyncAwait() {
```
Comment (735952) [inline portfolio/src/main/webapp/script.js:18]: you don't have to mention "asyncAwait" in the name of the method.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (andyp21) [reply portfolio/src/main/webapp/script.js:21]: Is this in regards to the two 'await' calls? - using async versus arrow function promise chain?
I saw both in the walkthrough example

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (andyp21) [reply portfolio/src/main/webapp/script.js:18]: No problem!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (735952) [reply portfolio/src/main/webapp/script.js:21]: No, github was showing "document.getElementById('greeting-container').innerText = greeting;" and the line below it was the same logic but in two lines.

**Pull request merged**